_std/_accessLevels.dm

```diff
@@ -42,46 +42,47 @@
 #define access_maxsec 37 // The HoS' armory.
 #define access_securitylockers 38
 #define access_carrypermit 39 // Are allowed to carry sidearms as far as guardbuddies and secbots are concerned.
-#define access_engineering 40 // General engineering area and substations.
-#define access_engineering_storage 41 // Main metal/tool storage things.
-#define access_engineering_eva 42 // Engineering space suits. Currently unused.
-#define access_engineering_power 43 // APCs and related supplies.
-#define access_engineering_engine 44 // Engine room.
-#define access_engineering_mechanic 45 // Electronics lab.
-#define access_engineering_atmos 46 // Engineering's supply of gas canisters.
-#define access_engineering_control 48 // Engine control room.
-#define access_engineering_chief 49 // CE's office.
+#define access_contrabandpermit 40 // Are allowed to handle contraband as far as guardbuddies and secbots are concerned.
+#define access_engineering 41 // General engineering area and substations.
+#define access_engineering_storage 42 // Main metal/tool storage things.
+#define access_engineering_eva 43 // Engineering space suits. Currently unused.
+#define access_engineering_power 44 // APCs and related supplies.
+#define access_engineering_engine 45 // Engine room.
+#define access_engineering_mechanic 46 // Electronics lab.
+#define access_engineering_atmos 47 // Engineering's supply of gas canisters.
+#define access_engineering_control 49 // Engine control room.
+#define access_engineering_chief 50 // CE's office.
```

Comment:
I don't like shifting other access ids unnecessarily. For example this will break saved custom jobs.

Reply:
Good point, reverted the shift, picked next available number.